cypress/e2e/3-default-owner.cy.ts

```diff
@@ -26,90 +26,92 @@ const lastName = randLastName();
-			credentialsModal.getters.newCredentialTypeSelect().should('be.visible');
-			credentialsModal.getters.newCredentialTypeOption('Notion API').click();
+		credentialsModal.getters.newCredentialModal().should('be.visible');
+		credentialsModal.getters.newCredentialTypeSelect().should('be.visible');
+		credentialsModal.getters.newCredentialTypeOption('Notion API').click();
 
-			credentialsModal.getters.newCredentialTypeButton().click();
+		credentialsModal.getters.newCredentialTypeButton().click();
 
-			credentialsModal.getters.connectionParameter('API Key').type('1234567890');
+		credentialsModal.getters.connectionParameter('API Key').type('1234567890');
 
-			credentialsModal.actions.setName('My awesome Notion account');
-			credentialsModal.actions.save();
+		credentialsModal.actions.setName('My awesome Notion account');
+		credentialsModal.actions.save();
 
-			credentialsModal.actions.close();
+		credentialsModal.actions.close();
 
-			credentialsModal.getters.newCredentialModal().should('not.exist');
-			credentialsModal.getters.editCredentialModal().should('not.exist');
+		credentialsModal.getters.newCredentialModal().should('not.exist');
+		credentialsModal.getters.editCredentialModal().should('not.exist');
 
-			credentialsPage.getters.credentialCards().should('have.length', 1);
-		});
+		credentialsPage.getters.credentialCards().should('have.length', 1);
+	});
 
-		describe('should be able to setup UM from settings', () => {
-			mainSidebar.getters.settings().should('be.visible');
-			mainSidebar.actions.goToSettings();
-			cy.url().should('include', settingsUsersPage.url);
+	it('should be able to setup UM from settings', () => {
+		mainSidebar.getters.settings().should('be.visible');
+		mainSidebar.actions.goToSettings();
+		cy.url().should('include', settingsUsersPage.url);
 
-			settingsUsersPage.actions.goToOwnerSetup();
+		settingsUsersPage.actions.goToOwnerSetup();
 
-			cy.url().should('include', signupPage.url);
-		});
+		cy.url().should('include', signupPage.url);
+	});
 
-		describe('should be able to setup instance and migrate workflows and credentials', () => {
-			cy.setup({ email, firstName, lastName, password });
+	it('should be able to setup instance and migrate workflows and credentials', () => {
+		cy.setup({ email, firstName, lastName, password });
 
-			messageBox.getters.content().should('contain.text', '1 existing workflow and 1 credential')
+		messageBox.getters.content().should('contain.text', '1 existing workflow and 1 credential')
 
-			messageBox.actions.confirm();
-		});
+		messageBox.actions.confirm();
+		cy.url().should('include', settingsUsersPage.url);
+	});
 
-		describe('should be redirected back to users page after setup', () => {
-			cy.url().should('include', settingsUsersPage.url);
-			// todo test users and that owner exist
-		});
+	it('can click back to workflows and have migrated workflow after setup', () => {
+		cy.signin({ email, password });
```

Comment:
preferred this in one test wihtout sign in.. to ensure user is logged in directly after setup and redirected successfully and that migration is complete..

Reply:
Alright, I've merged this spec with the previous one